itest/src/scala33x/build.sc

```diff
@@ -0,0 +1,25 @@
+import $file.plugins
+import io.github.davidgregory084.TpolecatModule
+import mill._
+import mill.scalalib.ScalaModule
+
+object project extends ScalaModule with TpolecatModule {
+  override def scalaVersion = "3.3.3"
+}
+
+def verify() = T.command {
+  assert(project.scalacOptions() == Seq(
+    "-encoding",
+    "utf8",
+    "-deprecation",
+    "-explain-types",
+    "-explain",
+    "-feature",
+    "-language:experimental.macros",
+    "-language:higherKinds",
+    "-language:implicitConversions",
+    "-unchecked",
+    "-Xfatal-warnings",
+    "-Ykind-projector"
+  ), s"scalacOptions did not match, got instead: ${project.scalacOptions()}")
+}
```

Comment:
I expected this test to fail, but it didn't (locally), which leads me to believe it isn't running. How should I be running it?